### PR TITLE
edits to the descriptive text on VT landing page

### PR DESCRIPTION
**Pull request by laurensorensen**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -44,11 +44,16 @@ en:
...
+<li><a href="https://exhibits.stanford.edu/virtual-tribunals/feature/world-war-ii-u-s-army-courts-europe">World War II U.S. Army Courts, Europe</a></li>
+<li><a href="https://exhibits.stanford.edu/virtual-tribunals/feature/world-war-ii-u-s-army-courts-asia-pacific">World War II U.S. Army Courts, Asia-Pacific</a></li>
+<li><a href="https://exhibits.stanford.edu/virtual-tribunals/feature/special-panels-for-serious-crimes-east-timor-spsc">Special Panels for Serious Crimes in East Timor (SPSC)</a></li>
+<li><a href="https://exhibits.stanford.edu/virtual-tribunals/feature/taube-archive-of-the-international-military-tribunal-imt-at-nuremberg-1945-46">Digitized static documents from the Nuremberg Trial Archives</a></li>
+</ul> are included for browsing and searching in the Spotlight platform. For searching this particular platform, please refer to the <a href="https://exhibits.stanford.edu/virtual-tribunals/about/how-to-search-the-documents">How to Search the Collections page</a> for tips on searching using keywords and facets, and how to search across collections within Spotlight.</p>
```
Comment (corylown) [inline config/locales/en.yml:58]: @laurensorensen this proposed change removes the line breaks so the app can load the yml file.
```suggestion
            exhibit_description_html: <p>At present, Virtual Tribunals collections are hosted on two different platforms. The <a href="http://exhibits.stanford.edu/virtual-tribunals">Virtual Tribunals exhibit</a> provides access through the open source <a href="https://library.stanford.edu/spotlight-digital-exhibits">Spotlight</a> software. Trial records from <ul><li><a href="https://exhibits.stanford.edu/virtual-tribunals/feature/world-war-ii-u-s-army-courts-europe">World War II U.S. Army Courts, Europe</a></li><li><a href="https://exhibits.stanford.edu/virtual-tribunals/feature/world-war-ii-u-s-army-courts-asia-pacific">World War II U.S. Army Courts, Asia-Pacific</a></li><li><a href="https://exhibits.stanford.edu/virtual-tribunals/feature/special-panels-for-serious-crimes-east-timor-spsc">Special Panels for Serious Crimes in East Timor (SPSC)</a></li><li><a href="https://exhibits.stanford.edu/virtual-tribunals/feature/taube-archive-of-the-international-military-tribunal-imt-at-nuremberg-1945-46">Digitized static documents from the Nuremberg Trial Archives</a></li></ul> are included for browsing and searching in the Spotlight platform. For searching this particular platform, please refer to the <a href="https://exhibits.stanford.edu/virtual-tribunals/about/how-to-search-the-documents">How to Search the Collections page</a> for tips on searching using keywords and facets, and how to search across collections within Spotlight.</p>
```

**Review by corylown — CHANGES_REQUESTED** [review]
@laurensorensen let me know if you want to pair on getting this pull request to pass the build/test so we can merge it.

File: config/locales/en.yml (hunk 1 above)
Comment (laurensorensen) [reply config/locales/en.yml:58]: Thanks for doing that! I'm realizing that I jumped the gun a little bit on this. We're releasing the two new collections on Dec 10 (though they are technically live now for people on VT team to review). So I'm going to put on my calendar to come back to this on December 5. Does that sound ok?

Comment (corylown) [reply config/locales/en.yml]: @laurensorensen I put these changes up in a separate pull request, see: https://github.com/sul-dlss/vt-arclight/pull/601

**Pull request closed without merging**